apps/mull-ui/src/app/components/event-page-header/event-page-header.tsx

```diff
@@ -9,26 +9,16 @@ import './event-page-header.scss';
 import { IEvent } from '@mull/types';
 
 import MullBackButton from '../mull-back-button/mull-back-button';
+import { formatDate } from '../../../utilities';
 
 export interface EventPageHeaderProps {
   event: IEvent;
   prevPage: string;
 }
 
 export const EventPageHeader = ({ event, prevPage }: EventPageHeaderProps) => {
-  const dateToString = (date: Date): string => {
-    const dateString = Intl.DateTimeFormat('en-us', {
-      month: 'short',
-      day: 'numeric',
-    }).format(date);
-    const timeString = Intl.DateTimeFormat('en-us', {
-      hour: '2-digit',
-      minute: '2-digit',
-    }).format(date);
-
-    return `${dateString}\n${timeString}`;
-  };
-
+  const { day: startDay, month: startMonth, time: startTime } = formatDate(event.startDate);
+  const { day: endDay, month: endMonth, time: endTime } = formatDate(event.endDate);
   return (
     <div className="event-page-header">
       <MullBackButton>{prevPage}</MullBackButton>
```

Comment:
This is not related to your ticket but I just noticed that pressing the back button reloads the website instead of going to the previous page. Would be worth creating a bug ticket for it.

Reply:
Good point, will create one